pkg/plugins/policies/core/matchers/egress.go

```diff
@@ -35,11 +35,20 @@ func EgressMatchedPolicies(rType core_model.ResourceType, tags map[string]string
 
 	var fr core_rules.FromRules
 	var err error
-	if isFrom {
+
+	// in case the policy support
+	switch {
+	case isFrom && isTo:
+		// we intentionally choose From
+		fr, err = processFromRules(tags, policies)
+	case isFrom:
+		// choose From
```

Comment:
nit: I think we can remove `choose From/To` comments because they're pretty self-explanatory